### PR TITLE
Fix typo in confirmation email [BB-2229]

### PR DESCRIPTION
**Pull request by alanoe**
Replace "bring" with "begin" in user account confirmation email text. Old word makes no sense in the sentence where it was.

### CODE REVIEW

**Comment by viadanna** [comment]
@alanoe The change looks good, can you add a PR description?

I've merged master to update the branch and also get the tests running again.

**Comment by alanoe** [comment]
@viadanna Added a description to the PR. integration_tests_group_2 tests failed with a timeout, but apparently it has nothing to do with this change.

File: email_verification/templates/email_verification/verify.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
                     as well to continue.
                 </p>
                 <p>
-                    Once your emails are verified, we will bring preparing your new instance and you
+                    Once your emails are verified, we will begin preparing your new instance and you
```
Comment (viadanna) [inline email_verification/templates/email_verification/verify.html]: 👍

**Comment by alanoe** [comment]
@viadanna Thanks! I cannot merge, probably due to unrelated connection issue which happened in integration_tests_group_1 test group. Can someone force the merge or should I do something here?

**Comment by viadanna** [comment]
Let's rerun the failed one

**Comment by alanoe** [comment]
@viadanna Thanks for running the tests again. Unfortunately, we got the same error when provisioning an app server.

**Pull request merged**